feat: display selected count and size on action bar

sabersalv opened the pull request:
<!--- Provide a general summary of your changes in the Title above -->

## Description

Will show up only if number of the selected items are larger than or equal to 2.

## Screenshots

<img width="1116" alt="Screen Shot 2021-08-02 at 11 59 47 PM" src="https://user-images.githubusercontent.com/2525544/127891102-1528e0e2-0c3a-4cbc-8fa4-5daaf104693b.png">
<img width="426" alt="Screen Shot 2021-08-03 at 12 00 49 AM" src="https://user-images.githubusercontent.com/2525544/127891111-32b7ae07-81c1-46c2-961d-cfc5432fad54.png">
<img width="320" alt="Screen Shot 2021-08-03 at 12 01 00 AM" src="https://user-images.githubusercontent.com/2525544/127891113-557e1e18-ad9e-4cd2-b4e9-39506117f0e7.png">
<img width="1132" alt="Screen Shot 2021-08-03 at 12 04 11 AM" src="https://user-images.githubusercontent.com/2525544/127891116-436f5d12-5fba-4c99-8f56-319ba8bef277.png">
<img width="429" alt="Screen Shot 2021-08-03 at 12 04 34 AM" src="https://user-images.githubusercontent.com/2525544/127891117-be4b0b3c-e50b-49b4-afb5-265afd424990.png">
<img width="320" alt="Screen Shot 2021-08-03 at 12 04 41 AM" src="https://user-images.githubusercontent.com/2525544/127891118-eb2090e0-841b-404c-9d35-258ab1a59ba7.png">

## Types of changes

<!--- What types of changes does your code introduce? -->

- [ ] Breaking change (changes that break backward compatibility of public API or CLI - semver MAJOR)
- [x] New feature (non-breaking change which adds functionality - semver MINOR)
- [ ] Bug fix (non-breaking change which fixes an issue - semver PATCH)

jesec commented:
Hmm.... That does not look good on smaller screens. We should probably hide it when screen width is smaller than a certain value. 

It also feels weird on expanded view. The size should scale to match other components on the bar.

sabersalv commented:
Fixed

<img width="305" alt="Screen Shot 2021-08-06 at 12 10 21 AM" src="https://user-images.githubusercontent.com/2525544/128384165-d7f0fb3a-53c0-4bb2-a031-c02a7f858995.png">

Hide the count for screens smaller than 375px. I was trying to add mulit select for mobile, found out it's too complicated. I added a [select all button](https://github.com/jesec/flood/pull/386) instead. Mobile can still filter the torrents by sidebar (for example, view count with a tracker + a tag)

sabersalv commented:
Closed until you can think of a better design for this.
